### PR TITLE
Add Chinese translation 

### PR DESCRIPTION
**Pull request by dashinfantry**

### CODE REVIEW

**Comment by piggz** [comment]
This needs to be in a new file suffixed with -zh i think

**Comment by dashinfantry** [comment]
Chinese is"zh_cn"

**Comment by 0312birdzhang** [comment]
You should add a new file  `translations/harbour-advanced-camera-zh_CN.ts`

File: translations/harbour-advanced-camera-fr.ts [hunk 1]
```diff
@@ -1,128 +1,263 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.1" language="fr_FR">
+<TS version="2.1" language="zh_cn">
```
Comment (0312birdzhang) [inline translations/harbour-advanced-camera-fr.ts]: it should be `<TS version="2.1" language="zh_CN">`

**Pull request closed without merging**